python/cuvs/cuvs/preprocessing/quantize/binary/binary.pyx

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# cython: language_level=3
+
+import numpy as np
+
+from cuvs.common cimport cydlpack
+
+from pylibraft.common import auto_convert_output, device_ndarray
+from pylibraft.common.cai_wrapper import wrap_array
+
+from cuvs.common.exceptions import check_cuvs
+from cuvs.common.resources import auto_sync_resources
+from cuvs.neighbors.common import _check_input_array
+
+
+@auto_sync_resources
+@auto_convert_output
+def transform(dataset, output=None, resources=None):
```

Comment:
Can we add a note here about the bitwise_hamming distance as well?

Reply:
https://github.com/rapidsai/cuvs/pull/660/commits/11d16ee36e1ca53f91f3312ef06ca67c63fe1301